### PR TITLE
Added lognormal distribution

### PR DESCRIPTION
**Pull request by fonnesbeck**
Included a test that works for everything but `check_int_to_1`, so I added a `False` argument to the test call. Not sure why this is -- I confirmed that I have the correct log-likelihood manually.

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -231,6 +231,54 @@ def logp(value):
 
 
 @tensordist(continuous)
+def Lognormal(mu=0, tau=1):
```
Comment (jsalvatier) [inline pymc/distributions/continuous.py:234]: Should it be called Lognormal or LogNormal? I don't have a strong preference

**Comment by jsalvatier** [comment]
Looks good, merge when it passes tests.

**Comment by fonnesbeck** [comment]
I noticed that `Tpos` also excludes the integration test. Have you had problems with the test as well?

File: pymc/distributions/continuous.py (hunk 1 above)
Comment (fonnesbeck) [inline pymc/distributions/continuous.py:234]: I left it as `Lognormal` for consistency with PyMC 2. No preference otherwise.

**Pull request merged**